python/samples/06.auth.oauth.bot/README.md

```diff
@@ -0,0 +1,60 @@
+This sample shows how to incorporate a basic conversational SSO flow into a Microsoft Teams application using [Bot Framework](https://dev.botframework.com) and the Teams AI SDK.
+
+This sample requires creating an OAuth Connection in Azure Bot Service, which provides a token store to store the token after sign-in.
+
+Note that this bot will only work in tenants where the following graph scopes are permitted:
+
+-   `User.Read`
+
+<!-- @import "[TOC]" {cmd="toc" depthFrom=1 depthTo=6 orderedList=false} -->
+
+<!-- code_chunk_output -->
+
+-   [OAuth Bot](#oauth-bot)
+    -   [Interacting with the bot](#interacting-with-the-bot)
+    -   [Setting up the sample](#setting-up-the-sample)
+    -   [Testing the sample](#testing-the-sample)
+        -   [Using Teams Toolkit for Visual Studio Code](#using-teams-toolkit-for-visual-studio-code)
+
+<!-- /code_chunk_output -->
+
+## Interacting with the bot
+
+You can interact with this bot by sending it a message, which will echo back to you.
+
+
+## Setting up the sample
+
+1. Clone the repository
+
+    ```bash
+    git clone https://github.com/Microsoft/teams-ai.git
+    ```
+
+1. Follow below instructions to run the sample
+
+The easiest and fastest way to get up and running is with Teams Toolkit as your development guide. To use Teams Toolkit to continue setup and debugging, please continue below. To read about other options, skip to [Other ways to run the sample](#other-ways-to-run-the-sample).
+
+## Testing the sample
```

Comment:
also need to force add the `/env` folder to both samples

in particular, these are the keys I needed in my .env.local.user file for this bot
```
BOT_ID=
TEAMS_APP_ID=
BOT_DOMAIN=
BOT_ENDPOINT=
TEAMS_APP_TENANT_ID=
RESOURCE_SUFFIX=
AZURE_SUBSCRIPTION_ID=
AZURE_RESOURCE_GROUP_NAME=
AAD_APP_OBJECT_ID=
AAD_APP_TENANT_ID=
AAD_APP_OAUTH_AUTHORITY=
AAD_APP_OAUTH_AUTHORITY_HOST=
AAD_APP_ACCESS_AS_USER_PERMISSION_ID=


```